src/net/utils/manifest_loader.ts

```diff
@@ -47,10 +48,12 @@ const manifestPreLoader = (
   options: {
     customManifestLoader?: CustomManifestLoader;
     ignoreProgressEvents?: true;
-  }) => (url: string) : ILoaderObservable<Document|string> => {
+  }) => (
+    url: string, responseType?: "text"|"document"
+  ) : ILoaderObservable<Document|string> => {
     const { customManifestLoader, ignoreProgressEvents } = options;
     if (!customManifestLoader) {
-      return regularManifestLoader(url, ignoreProgressEvents);
+      return regularManifestLoader(url, ignoreProgressEvents, responseType);
```

Comment:
You forgot to not call the ``customManifestLoader`` if we're loading a sub-part.